backend/internal/model/game.go

```diff
@@ -2,7 +2,17 @@ package model
 
 import "errors"
 
-type Game struct {
+const (
+	teamWhite int = 1
+	teamBlue  int = 2
```

Comment:
```suggestion
	teamWhite = 1
	teamBlue   = 2
```

`int` is the default type. If you _want_ to specify type, only specify it for the first const

Reply:
ofc this is fine too